File: DCAnalyticsWebShell/ClientApp/src/models/Certification.ts
import { Section } from "./Section";
import { Statuses, CertificationTypes } from "../enums/enums";

export class Certification {
  public Key: string;
  public Status: Statuses;
  public Sequence: number;
  public Name: string;
  public CreatedBy: string
  public DateTime: Date;
  public Description: string;
  public CerificationType: CertificationTypes;
  public Sections: Section[];
  public FarmerKey: string;
  public ConfigurationId: string;
  public Deleted : boolean
}

File: DCAnalyticsMobile/DCAnalyticsMobile/Models/Certification.cs
using DCAnalytics;
using System;
using System.Collections.Generic;
using System.Text;

namespace DCAnalyticsMobile.Models
{
    public class Certification: DataCollectionObject
    {
        public Statuses Status { get; set; }
        public int Sequence { get; set; }
        public DateTime DateTime { get; set; }
        public string Description { get; set; }
        public CertificationTypes CerificationType { get; set; }
        public List<Section> Sections { get; set; }
        public string FarmerKey { get; set; }
        public string Template { get; set; }
        public int ConfigurationId { get; set; }
    }
}
